buildSrc/src/main/kotlin/GeneratePluginDocsTask.kt

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright (C) 2025 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+import groovy.json.JsonSlurper
+
+import org.gradle.api.DefaultTask
+import org.gradle.api.file.ConfigurableFileTree
+import org.gradle.api.tasks.InputFiles
+import org.gradle.api.tasks.Internal
+import org.gradle.api.tasks.OutputDirectory
+import org.gradle.api.tasks.TaskAction
+
+abstract class GeneratePluginDocsTask : DefaultTask() {
+    init {
+        group = "Documentation"
+        description = "Generate documentation for the plugins."
+    }
+
+    @get:InputFiles
+    abstract var inputFiles: ConfigurableFileTree
+
+    @OutputDirectory
+    val outputDirectory = project.layout.projectDirectory.file("website/docs/plugins").asFile
+
+    @Internal
+    val jsonSlurper = JsonSlurper()
+
+    @TaskAction
+    fun generatePluginDocs() {
+        require(inputFiles.files.isNotEmpty()) {
```

Comment:
Ideally, this should `throw GradleException("...")` instead.

Reply:
Is this documented somewhere? I couldn't find it.

Reply:
I'll do this as part of my PR then.